atomic_reactor/schemas/config.json

```diff
@@ -441,6 +441,10 @@
         "description": "Proxy to access yum repositories",
         "type": "string"
     },
+    "allow_external_base_image": {
```

Comment:
maybe "skip_koji_check_for_base_image" ?

external base image sound like we are using some image from some external repo

Reply:
changed